src/shadowbox/server/manager_metrics.ts

```diff
@@ -15,8 +15,24 @@
 import {PrometheusClient} from '../infrastructure/prometheus_scraper';
 import {DataUsageByUser, DataUsageTimeframe} from '../model/metrics';
 
+interface TunnelTimeDuration {
+  seconds: number;
+}
+
+interface TunnelTimeRequest {
+  time_window: TunnelTimeDuration;
+}
+
+interface TunnelTimeResponse {
```

Comment:
The naming here is a bit off. A request has only one response. The TunnelTime response is actually a list of entries.

Perhaps call this TunnelTimeEntry?